AsyncDisplayKit/ASVideoNode.mm

```diff
@@ -58,8 +65,6 @@ - (void)interfaceStateDidChange:(ASInterfaceState)newState fromState:(ASInterfac
       if (_shouldBePlaying) {
         [self pause];
         _shouldBePlaying = YES;
```

Comment:
Ah yeah, @appleguy  this is a weird thing i need to change but didn't want to until i had time to put a test or two around the logic.  I reset it to _shouldBePlaying because a call to -pause sets shouldBePlaying to NO.  Probably a side effect i need to get rid of, but I was originally trying to cut down on duplication of setting that property, lol